packages/state-manager/src/sagas/identity/savedOwnerCertificate/savedOwnerCertificate.saga.ts

```diff
@@ -10,26 +10,5 @@ export function* savedOwnerCertificateSaga(
   socket: Socket,
   action: PayloadAction<ReturnType<typeof identityActions.savedOwnerCertificate>['payload']>
 ): Generator {
-  let communityId: string = action.payload
-
-  if (!communityId) {
-    communityId = yield* select(communitiesSelectors.currentCommunityId)
-  }
-
-  const community = yield* select(communitiesSelectors.selectById(communityId))
-  const identity = yield* select(identitySelectors.selectById(communityId))
```

Comment:
this is moved to registerCertificateSaga for what reason? i don't see any problem but i'm curious!

Reply:
Yea, so we can create the community (setup the storage service and the registration service) and then register the owner's certificate.